js/screens/SignUp/SignUpScreen.js

```diff
@@ -48,38 +44,26 @@ class SignUpScreen extends Component {
   };
 
   addUser() {
-    const validEmail = /^([\w-.]+)@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.)|(([\w-]+\.)+))([a-zA-Z]{2,4}|[0-9]{1,3})(\]?)$/;
     const errorMessage = {
       code: 'Invalid email or password',
       message:
         'Please Enter a valid email and password of at least 6 characters'
     };
     const userMessage = {
```

Comment:
I do not think you have to create an object to show an error. You can just pass a string in your function and have the same result.